doc/memos/rdm0.md

```diff
@@ -0,0 +1,126 @@
+RIOT Maintainers Community (RIOT-MC)  
+Developer Memo: 0  
+Author(s): Emmanuel Baccelli  
+Date: December 2016
+
+***
+RIOT Developer Memo Format, Publishing and Maintenance Process
```

Comment:
(Just saying: looks weird, that it the title of the document is smaller than the section titles below in rendered Markdown)